### PR TITLE
[next] tree: promote changes from next-devel at 858cfe545d4e17abc751826495880437e00d9390

### PR DESCRIPTION
**Pull request by coreosbot-releng**
Triggered by @sohankunkerkar in https://github.com/coreos/fedora-coreos-streams/issues/296.

### CODE REVIEW

File: manifest-lock.x86_64.json [hunk 1]
```diff
@@ -336,9 +336,6 @@
     "fuse3-libs": {
       "evra": "3.10.2-1.fc34.x86_64"
     },
-    "fwupd": {
-      "evra": "1.5.8-1.fc34.x86_64"
-    },
```
Comment (dustymabe) [inline manifest-lock.x86_64.json]: hmm. We probably didn't intend for this to happen. We might need to add this back.

Comment (jlebon) [reply manifest-lock.x86_64.json]: Nice catch on this BTW! Hmm, maybe we should have CI also fail on package removals to be safe.

Anyway, fix in https://github.com/coreos/fedora-coreos-config/pull/985.

**Review by jlebon — APPROVED** [review]

**Comment by sohankunkerkar** [comment]
CI failing on `RPM diff`
```
[2021-04-27T18:15:55.418Z] Downgraded:

[2021-04-27T18:15:55.418Z]   crun 0.19-2.fc34 -> 0.19.1-2.fc34

[2021-04-27T18:15:55.418Z]   shim-x64 15-8 -> 15.4-4

```

**Comment by jlebon** [comment]
> CI failing on `RPM diff`

Yeah, all the packages downgraded in that list are hitting https://github.com/coreos/rpm-ostree/issues/2668, so we can ignore it.

**Pull request merged**